x-pack/plugin/vector-tile/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/vectortile/VectorTileCCSIT.java

```diff
@@ -69,19 +69,31 @@ public void testBasic() throws IOException {
         try (RestClient local = buildLocalClusterClient(); RestClient remote = buildRemoteClusterClient()) {
             final int localGeometries = createIndex(local, "test");
             final int remoteGeometries = createIndex(remote, "test");
-            // check call in each cluster
-            final Request mvtRequest = new Request(HttpPost.METHOD_NAME, "test/_mvt/location/0/0/0");
-            final VectorTile.Tile localTile = execute(local, mvtRequest);
-            assertThat(getLayer(localTile, "hits").getFeaturesCount(), Matchers.equalTo(localGeometries));
-            final VectorTile.Tile remoteTile = execute(remote, mvtRequest);
-            assertThat(getLayer(remoteTile, "hits").getFeaturesCount(), Matchers.equalTo(remoteGeometries));
-            // call to both clusters
-            final Request mvtCCSRequest = new Request(HttpPost.METHOD_NAME, "/test,other:test/_mvt/location/0/0/0");
-            final VectorTile.Tile ccsTile = execute(local, mvtCCSRequest);
-            assertThat(getLayer(ccsTile, "hits").getFeaturesCount(), Matchers.equalTo(localGeometries + remoteGeometries));
+            // check with no params
+            assertLocalAndRemote(local, remote, localGeometries, remoteGeometries, "");
+            // check with labels
+            assertLocalAndRemote(local, remote, 2 * localGeometries, 2 * remoteGeometries, "?with_labels=true");
```

Comment:
Why are there double the geometries when we ask for labels?

Reply:
Because when using labels we are add a extra feature with label information.